src/app/components/SocialLogin/SocialLogin.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './SocialLogin.module.css';
+
+type SocialLoginProps = {
+  icon: React.SVGProps<SVGSVGElement>;
```

Comment:
You could be less specific here to allow `img` elements too:
```suggestion
  icon: JSX.Element;
```